third-party/qthread/qthread-1.10/src/interfaces/chapel/tasks-qthreads.h

```diff
@@ -307,14 +309,17 @@ void chpl_task_stdModulesInitialized(void);
 // Wrap qthread_get_tasklocal() and assert that it is always available.
 static inline chpl_qthread_tls_t * chpl_qthread_get_tasklocal(void)
 {
-    chpl_qthread_tls_t * tls;
+    chpl_qthread_tls_t* tls;
 
     if (chpl_qthread_done_initializing) {
         tls = (chpl_qthread_tls_t *)
               qthread_get_tasklocal(sizeof(chpl_qthread_tls_t));
         if (tls == NULL &&
             pthread_equal(pthread_self(), chpl_qthread_comm_pthread))
-          tls = &chpl_qthread_comm_task_tls;
+            tls = &chpl_qthread_comm_task_tls;
```

Comment:
Is it worth changing this to:

 if (tls == NULL) {
  if (pthread_equal(pthread_self(), chpl_qthread_comm_pthread))
    tls = &chpl_qthread_comm_task_tls;
  else if (pthread_equal(pthread_self(), chpl_qthread_process_pthread))
    tls = &chpl_qthread_process_tls;
}

to eliminate the second NULL comparison?